flowexperimental/comp/wells/CompWellEquations_impl.hpp

```diff
@@ -17,6 +17,14 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+#ifndef OPM_COMP_WELL_EQUATIONS_IMPL_HPP
+#define OPM_COMP_WELL_EQUATIONS_IMPL_HPP
+
+#ifndef OPM_COMP_WELL_EQUATIONS_HPP
+#include <config.h>
```

Comment:
Why is this a good idea? It is a header. Maybe add a comment.

This header does not get installed, though. It is listed in MAIN_SOURCE_FILES in CmakeList_files.cmake. Quite confusing.

Reply:
How can I "fix my IDE". The IDE processes the currently processed file as a separate compile unit. That means symbols are not defined. It is a good idea because when the _impl.hpp is processed by the IDE (ie the LSP) it knows the symbols. In normal compilation this does nothing because the preprocessor token is defined.

Reply:
Added the comment used elsewhere.